test/com/stanimiriliev/javaobjects/sumator/SumatorTest.java

```diff
@@ -0,0 +1,36 @@
+package com.stanimiriliev.javaobjects.sumator;
+
+import static org.junit.jupiter.api.Assertions.*;
+import org.junit.jupiter.api.Test;
+
+
+/**
+ * @author Stanimir Iliev
+ * @since 29.07.2017
+ */
+class SumatorTest {
+    
+    @Test
+    void testWithNormalEntry() {
+        assertEquals(19, Sumator.sum("15", "4"));
+    }
+
+    @Test
+    void testWithInvalidEntry(){
+        try{
+            Sumator.sum("a", "b");
+            fail("The method should throw NumberFormatException");
+        }
+        catch(NumberFormatException e){}
+    }
+
+    @Test
+    void testWithNullValue(){
+        try{
+            Sumator.sum(null, "b");
+            fail("The method should throw IllegalArgumentException");
+        }
+        catch(IllegalArgumentException e){}
+    }
```

Comment:
yes. Try it, of you do not throw exception, the test will fail. 
but more elegant is 
```java
@Test(expected = NumberFormatException.class)
void invalidArgument(){
  new Sumator().sum("A",7);
}
```